test_scripts/Defects/5_0/1877_SDL_does_not_ignore_invalid_request_response_from_hmi_and_does_not_log_internal_error.lua

```diff
@@ -0,0 +1,44 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/1877
+--
+-- Description:
+-- SDL does not ignore invalid request/response from hmi and does not log internal error
+-- Precondition:
+-- 1) Core, HMI started.
+-- 2) App is registered on HMI.
+-- In case:
+-- 1) HMI sends ActivateApp with fake param and invalid appID param
+-- Expected result:
+-- 1) SDL cuts off fake parameters, log the corresponding error and ignore this response (request)
+-- Actual result:
+-- N/V.
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('user_modules/sequences/actions')
+local runner = require('user_modules/script_runner')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Functions ]]
+local function activateApp(pAppId)
+    if not pAppId then pAppId = 1 end
+    local requestId = common.getHMIConnection():SendRequest("SDL.ActivateApp", { appID = 12345, fakeParam = "fakeParam" })
```

Comment:
According to defect description the appId must bw with invalid value, please update.